src/components/dataset/schemaCreation/DatasetSchemaInformationView.tsx

```diff
@@ -0,0 +1,334 @@
+      </Grid>
+
+      <Grid item xs={6} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-region"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.region })}
+        >
+          Region*
+        </label>
+        <Controller
+          name="region"
+          control={control}
+          rules={{ required: 'region is required' }}
+          render={({ field }) => (
+            <Autocomplete
+              id="dataset-region"
+              freeSolo
+              options={regionOptions}
+              className={classes.formInput}
+              renderInput={(params: any) => (
+                <TextField
+                  {...params}
+                  error={!!errors.region}
+                  helperText={errors.region ? errors.region.message : ''}
+                />
+              )}
+              {...field}
+              onChange={(_event: any, change: any) => {
+                field.onChange(change);
+              }}
+            />
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-stewards"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.stewards })}
+        >
+          Stewards
+        </label>
+        <div className={classes.formFieldDescription}>
+          A Steward, or Data Owner, is the person who created the dataset. While they are ultimately
+          liable for the data, they can assign the hands-on data management to another person by
+          assigning the Custodian role.
+        </div>
+        <Controller
+          name="stewards"
+          control={control}
+          rules={{
+            validate: {
+              isEmail: (values: string[]) => {
+                const emailErrors = _.filter(values, (v: string) => !isEmail(v));
+                return emailErrors.length === 0 || `Invalid emails: "${emailErrors.join('", "')}"`;
+              },
+            },
+          }}
+          render={({ field }) => (
+            <Autocomplete
+              id="dataset-stewards"
+              freeSolo
+              multiple
+              options={[]}
+              className={classes.formInput}
+              renderInput={(params: any) => (
+                <TextField
+                  {...params}
+                  error={!!errors.stewards}
+                  helperText={errors.stewards ? errors.stewards.message : ''}
+                />
+              )}
+              {...field}
+              onChange={(_event: any, change: any) => {
+                field.onChange(change);
+              }}
+            />
+          )}
+        />
+      </Grid>
+
+      <Grid item xs={12} className={classes.formFieldContainer}>
+        <label
+          htmlFor="dataset-custodians"
+          className={clsx(classes.formLabel, { [classes.formLabelError]: errors.custodians })}
+        >
+          Custodians*
```

Comment:
the * means the field is required to have a value

Reply:
TOL: I wonder if that would be doable with a subcomponent that takes a prop named `required` and automatically adds that `*`. It feels a little strange to put that in the text, because I think it could easily lead to inconsistencies in the way required fields are handled across the UI.